insight.py

```diff
@@ -51,7 +53,7 @@ def __init__(self, args):
         if not args.alias:
             self.alias = util.get_hostname()
         self.full_outdir = fileutils.create_dir(
-            os.path.join(self.outdir, self.alias))
+            os.path.join(self.cwd, self.outdir, self.alias))
```

Comment:
What if a absolute path is passed in with `--output`? Maybe we can check with `str.startswith('/')`?